public/apps/access-error-component.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ *   Copyright OpenSearch Contributors
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import { EuiPageContent } from '@elastic/eui';
+import React from 'react';
+
+interface AccessErrorComponentProps {
+  dataSourceLabel?: string;
+  message?: string;
+}
+
+export const AccessErrorComponent: React.FC<AccessErrorComponentProps> = (props) => {
```

Comment:
Will this page also be rendered when MDS is disabled and a user tries to directly navigate to a security URL without having the proper access?

i.e. if a user goes directly to `http://localhost:5601/app/security-dashboards-plugin#/users` by typing it into the URL instead of navigating with the menu.

When I try to directly go to a security url as a non-admin I currently get Application Not Found.

Reply:
we do not register security plugin if either MDS is not enabled or user doesn't have sufficient permissions. Hence, Application not found will still be displayed in this case where MDS is disabled.